src/Core/src/Hosting/LifecycleEvents/AppHostBuilderExtensions.Android.cs

```diff
@@ -76,7 +78,13 @@ static void OnConfigureWindow(IAndroidLifecycleBuilder android)
 				{
 					if (IPlatformApplication.Current is IPlatformApplication platformApplication)
 					{
-						platformApplication.Application?.ThemeChanged();
+						var application = platformApplication.Application;
+
+						if (application is not null)
+						{
+							application.UpdateNightMode();
+							application.ThemeChanged();
+						}
```

Comment:
Since they are methods, why not use:
```suggestion
						application?.UpdateNightMode();
						application?.ThemeChanged();
```

If I am not misspend the IL with the if will add a extra bool to the stack, but honestly FELL FREE TO IGNORE this comment.

Reply:
You are right. Changed.